docs/xdr/features/investigate/notebooks.md

```diff
@@ -0,0 +1,163 @@
+2. Click `New Notebook`
+3. Edit blank notebook or use an existing template
+
+![notebook_blank](/assets/operation_center/notebook_case.png){: style="max-width:100%"}
+
+!!! Warning
+    In a **multi-tenant** environment, notebooks are created in the **parent** workspace by default. They are not visible to child tenants unless you explicitly share the notebook with them.
+
+### Basic Features
+* Markdown syntax support
+* Image attachment for evidence collection
+* Multi datasources querying
+* Investigation findings documentation
+* Automations execution
+* Real-time modifications saving
+
+!!! Info
+    Currently, Markdown table syntax is not supported.
+
+### Tracking Changes
+* Last modification date
+* Last modification author
+
+![notebook_changes](/assets/operation_center/notebook_changes.png){: style="max-width:100%"}
+
+## Notebook Templates
+
+### Template Management
+Enhance incident response capabilities by creating template notebooks tailored to different threat types:
+
+* Phishing investigations
+* Malware analysis
+* Intrusion containment
+* Any custom threat responses
+
+### Template Features
+* Pre-defined investigation steps
+* Threat tailored query templates
+* Suggested remediation actions
+* Guidance for analysts
+
+### Setting Default Notebook Template For Cases/Alerts
+1. Navigate to Notebook templates
+2. Select desired template
+3. Set as `default` for cases and alerts
+
+The `default` notebook template will be suggested for use in cases and alerts, enabling the SOC team to standardize their response processes.
+
+![notebook_template](/assets/operation_center/notebook_template.gif){: style="max-width:100%"}
+
+### Template Creation Methods
+* Create notebook template from scratch
+* Convert existing notebook to template
+
+![notebook_convert_template](/assets/operation_center/notebook_convert_template.gif){: style="max-width:100%"}
+
+## Command Menu and Formatting
+
+### Accessing the Command Menu
+* Press the slash key (/) to open the Command menu
+* Access notebook capabilities
+
+![notebook_command](/assets/operation_center/notebook_command.gif){: style="max-width:100%"}
+
+!!! Tip
+    When the command menu is open, quickly search for an integration by typing its name.
+
+### Available Commands
+* Text formatting options
+* Query builder insertion
+* Automation insertion
+
+![notebook_menu](/assets/operation_center/notebook_menu.png){: style="max-width:100%"}
+
+### Query Builder Editor
+* Display data in tables
+* Create various data visualizations, including pie charts, line charts, bar charts, column charts, and numeric displays
+
+![notebook_viz](/assets/operation_center/notebook_viz.png){: style="max-width:100%"}
+
+## Sharing Notebooks
+
+### Export notebook as PDF
+
+Convert notebooks into PDF files to efficiently share incident, threat hunting, and KPI reports.
+
+![notebook_share](/assets/operation_center/notebook_pdf.gif){: style="max-width:100%"}
```

Comment:
Same for this .gif, try uploading it with a more zoomed screen (or using AMA cool tool for screenshots!)